### PR TITLE
Followup fixes for physical devices

### PR DESCRIPTION
**Pull request by Jupeyy**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by Jupeyy** [comment]
is it ok that i add another translation string?

File: src/engine/client/backend/vulkan/backend_vulkan.cpp [hunk 1]
```diff
@@ -1210,7 +1210,7 @@ class CCommandProcessorFragment_Vulkan : public CCommandProcessorFragment_GLBase
 			m_RecreateSwapChain = true;
 			break;
 		default:
-			m_ErrorHelper = "unknown error";
+			m_ErrorHelper = "unknown error ";
```
Comment (def-) [inline src/engine/client/backend/vulkan/backend_vulkan.cpp:1213]: space?

Comment (def-) [reply src/engine/client/backend/vulkan/backend_vulkan.cpp:1213]: ok, i see, we add something in the line below, maybe `unknown error: ` then

**Comment by def-** [comment]
> is it ok that i add another translation string?

ok

**Comment by Jupeyy** [comment]
ok then at least add the first commit, else we potentially drop legit computers

**Pull request merged**